Ugly_Number/Ugly_Number.c

```diff
@@ -0,0 +1,41 @@
+//Ugly numbers are numbers whose only prime factors are 2, 3 or 5.
+
+# include <stdio.h>
+
+int main()
+{
+    int num,flag=0;
+ 
```

Comment:
Can use a bool for flag

Reply:
@sridharjajoo Is it compulsory to use bool?

Reply:
it is a good practice as bool is specifically used for these situations

Reply:
@sridharjajoo I have done the required changes. Please review.